src/data/misc/restaurant.rs

```diff
@@ -0,0 +1,191 @@
+use rand::Rng;
+use wasm_bindgen::prelude::*;
+// ask about ### and ???
+// TODO Ask about generating a restuarant object?
+
+#[wasm_bindgen]
+pub fn name_prefix() -> String {
+	let mut rng = rand::thread_rng();
+	NAME_PREFIX[rng.gen_range(0..NAME_PREFIX_LEN)].to_string()
+}
+
+pub fn name_suffix() -> String {
+	let mut rng = rand::thread_rng();
+	NAME_SUFFIX[rng.gen_range(0..NAME_SUFFIX_LEN)].to_string()
+}
+
+pub fn restaurant_full_name() -> String {
+	let prefix = name_prefix();
+	let suffix = name_suffix();
+	return format!("{} {}", prefix, suffix);
+}
+
+pub fn restaurant_type() -> String {
+	let mut rng = rand::thread_rng();
+	RESTAURANT_TYPE[rng.gen_range(0..RESTAURANT_TYPE_LEN)].to_string()
+}
+
+pub fn restaurant_description() -> String {
+	let mut rng = rand::thread_rng();
+	RESTAURANT_DESCRIPTION[rng.gen_range(0..RESTAURANT_DESCRIPTION_LEN)].to_string()
+}
+
+pub fn restaurant_review() -> String {
+	let mut rng = rand::thread_rng();
+	RESTAURANT_REVIEW[rng.gen_range(0..RESTAURANT_REVIEW_LEN)].to_string()
+}
+
+pub fn generate_full_restaurant() -> String {
```

Comment:
This is fine, but I do think we should discuss how to handle custom formatted data. So there's a standard across the library.

Reply:
That was actually one of my questions I was going to ask. If we should return and object instead or something. Lets discuss this whenever you have some time